2024/political-donations-by-cycle/main.js

```diff
@@ -0,0 +1,487 @@
+    222757, 225186, 225449, 226043, 226236, 227407, 228084, 229508, 229575,
+    229674, 229991, 230266, 233928, 237624, 238067, 238130, 240228, 240732,
+    242498, 242680, 243051, 243412, 243971, 243987, 245505, 246664, 246938,
+    247104, 247705, 247990, 248557, 249985, 250881, 252722, 254046, 254181,
+    256003, 256569, 257290, 257394, 257762, 258429, 258602, 260389, 261594,
+    261736, 262079, 262546, 263984, 264348, 264512, 265537, 266772, 266887,
+    267255, 267359, 268152, 268350, 268392, 268545, 268685, 268763, 268938,
+    269523, 270476, 270952, 275582, 275778, 279225, 279354, 279925, 281844,
+    282533, 282758, 283164, 285487, 286890, 287524, 291268, 291580, 291616,
+    293910, 295770, 298697, 300505, 301141, 301631, 302372, 302774, 303325,
+    304956, 305112, 306260, 306541, 307018, 307310, 308051, 309861, 310937,
+    312668, 313059, 313423, 313953, 315064, 315247, 316447, 316903, 319205,
+    320804, 322152, 323716, 332566, 332678, 333090, 333741, 333998, 336322,
+    337559, 341105, 342365, 343593, 343690, 344532, 345917, 346963, 348888,
+    349690, 350056, 353306, 358089, 362266, 363273, 363877, 365399, 366834,
+    367939, 370286, 370778, 371549, 374848, 375493, 377717, 378469, 380020,
+    381017, 381951, 382299, 382852, 384544, 386465, 386734, 391623, 392606,
+    394616, 395255, 398523, 399350, 400045, 402992, 407991, 411868, 415521,
+    418656, 419690, 421490, 422838, 424543, 426919, 427493, 430164, 433021,
+    434480, 435191, 436798, 437661, 440653, 440834, 441085, 441736, 442339,
+    442489, 442897, 443147, 443296, 443340, 444010, 444675, 445467, 445914,
+    446477, 447041, 447562, 447950, 448277, 448895, 449240, 449683, 449908,
+    449914, 450352, 450482, 450663, 450663, 450692, 450793, 450796, 450806,
+    450871, 450881, 451166, 451244, 451268, 451346, 451527, 451631, 451733,
+    452005, 452248, 452373, 452893, 453517, 453972
+];
+const start2022 = Date.UTC(2020, 9, 27);
+
+yAxisLabels = [0, 295038, 384179, 453972, 1289469];
+
+color2016 = '#C16622';
+color2018 = '#800000';
+color2020 = '#155F83';
+color2022 = '#8A9045';
+
+Highcharts.setOptions({
+    lang: {
+        thousandsSep: ','
+    }
+});
+
+Highcharts.chart('chart-div1', {
+    // Setting default colors
+    colors: ['#C16622', '#800000', '#155F83', '#8A9045'],
+
+    // All code for your chart goes here
+    chart: {
+        type: 'spline'
+    },
+    title: {
+        text: 'Cumulative Donations by Cycle from 2016-2022',
+        align: 'center'
+    },
+    subtitle: {
+        text: 'The 2020 Election saw record breaking donations by UChicago professors and nationwide.',
+        align: 'center',
+        style: {
+            fontSize: '15px'
+        }
+    },
+    xAxis: {
+        type: 'datetime',
+        labels: {
+            format: '{value:%b %Y}'
+        },
+        tickInterval: 30 * 24 * 3600 * 1000 * 4
+    },
+    yAxis: {
+        title: {
+            text: ''
+        },
```

Comment:
You should be able to delete this part, I believe.

Reply:
this was a cheat to remove axis title but changing to `enabled: false` which i think is cleaner and more descriptive with same effect